modules/test/playwright/helpers/HeadlessAdminUserApiHelper.ts

```diff
@@ -187,6 +217,51 @@ export class HeadlessAdminUserApiHelper {
 		);
 	}
 
+	async postOrganizationRoleUserAccountAssociation(
```

Comment:
assignUserToOrganizationRole (like the assignUserToAccountBy... that already exists)